scripts/.eslintrc.json

```diff
@@ -1,7 +1,13 @@
 {
+  "env": {
+    "node": true
+  },
   "globals": {
     "__dirname": false,
     "__filname": false,
     "__moduleName": true
+  },
+  "rules": {
+    "no-console": 0
```

Comment:
Could we use the eslint "magic string" value of `"off"` here? It's clearer once we start mixing in `"warn"` and `"error"`: http://eslint.org/docs/user-guide/configuring#configuring-rules